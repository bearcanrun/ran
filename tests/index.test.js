/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import withData from '../pages/index';

describe('Index Snapshot', () => {
  it('should render correctly', () => {
    const component = renderer.create(<withData />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
