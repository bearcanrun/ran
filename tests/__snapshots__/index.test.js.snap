// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Snapshot should render correctly 1`] = `<withData />`;
